src/components/CurriculumComponents/OakComponentsKitchen/Fieldset/__snapshots__/Fieldset.test.tsx.snap

```diff
@@ -7,7 +7,7 @@ exports[`Fieldset render 1`] = `
       class="sc-ad0ad6f5-0 sc-3dab0ad0-0 bZaRlP eSpKwY"
     >
```

Comment:
<img src="https://media0.giphy.com/media/lkdH8FmImcGoylv3t3/giphy.gif"/>

Not sure why these snapshot tests have been failing, has happened on a couple of PRs